js/myparcel.js

```diff
@@ -860,106 +882,76 @@ MyParcel = {
      * Postal Code.
      *
      */
-
-    callDeliveryOptions: function () {
+    callDeliveryOptions: function() {
         MyParcel.showSpinner();
         MyParcel.clearPickUpLocations();
+        MyParcel.hideDelivery();
+        MyParcel.setAddressFromInputFields();
 
-        var cc = this.data.address.cc;
-        var postalCode = this.data.address.postalCode;
-        var number = this.data.address.number;
-        var city = this.data.address.city;
-        var cutoffTime = this.data.config.cutoffTime;
-        var mondayDeliveryActive = 0;
-
-        if (postalCode == '' || number == '') {
+        // Hide PostNL field if there is no address entered
+        if (MyParcel.data.address.postalCode == '' || MyParcel.data.address.number == '') {
+            MyParcel.hideSpinner();
             MyParcel.showMessage(
-                '<h3>Adresgegevens zijn niet ingevuld</h3>'
+                '<h3>' + MyParcel.data.config.addressNotFound + '</h3>'
             );
-        }
-        if (cc === "BE") {
-            var numberExtra = this.data.address.numberExtra;
-            var street = this.data.address.street;
-        }
-
-        if (numberExtra) {
-            number = number + numberExtra;
-        }
-
-        /* Don't call API unless both Postcode and House Number are set */
-        if (!number || !postalCode) {
-            MyParcel.showFallBackDelivery();
             return;
         }
 
         /* Check if the deliverydaysWindow == 0 and hide the select input*/
         this.deliveryDaysWindow = this.data.config.deliverydaysWindow;
 
-        if (this.deliveryDaysWindow <= 0) {
+        if (this.deliveryDaysWindow === '0') {
             this.deliveryDaysWindow = 1;
         }
 
-        if (this.data.config.allowMondayDelivery === true) {
```

Comment:
Waarom gooi je dit weg?

Reply:
Dit was waarschijnlijk per ongeluk en had alleen in BE weg/anders gemoeten